test/spec/spectest.cpp

```diff
@@ -46,34 +36,37 @@ using namespace WasmEdge;
 
 // Preprocessing for set up aliasing.
 void resolveRegister(std::map<std::string, std::string> &Alias,
-                     rapidjson::Value &CmdArray,
-                     rapidjson::Document::AllocatorType &Allocator) {
-  rapidjson::Value::ValueIterator ItMod = CmdArray.Begin();
-  for (rapidjson::Value::ValueIterator It = CmdArray.Begin();
-       It != CmdArray.End(); ++It) {
-    const auto CmdType = It->GetObject()["type"].Get<std::string>();
+                     simdjson::dom::array CmdArray) {
+  std::string_view OrgName;
+  uint64_t Mod;
+  for (simdjson::dom::object Obj : CmdArray) {
+    std::string_view CmdType = Obj["type"];
+    int flag = 0;
```

Comment:
Please follow the coding style of the variable naming.
Also, the `flag` doesn't provide any information about what the flag is recorded. Please use a meaningful naming.

Reply:
Ok, will do

Reply:
I have changed the variable names [here](https://github.com/WasmEdge/WasmEdge/pull/2659/commits/4020806f88ab6fa990867cae568026c0782518df#:~:text=uint64_t%20LastModLine%3B,%3D%20false%3B), and changed `flag` to bool `Replaced`. 
Please let me know if the names are acceptable.